bot/cogs/reminders.py

```diff
@@ -165,14 +168,18 @@ def __init__(self, bot: Bot):
             json={
                 'author': ctx.author.id,
                 'channel_id': ctx.message.channel.id,
+                'jump_url': ctx.message.jump_url,
```

Comment:
This won't work without making sure the API accepts a `jump_url` field by including it in the model. I've added the `Stalled` label until there's support for this in `site`, since otherwise the implementation of python-discord/site#258 will break our reminders feature.

Reply:
Only admin can add a `jump_url` in the model I suppose?

Reply:
No, anyone can make a PR to the `site` repository that adds such a field in the [`reminder` model](https://github.com/python-discord/site/blob/master/pydis_site/apps/api/models/bot/reminder.py). The `Meta` class of the reminder serializer probably also needs to be adjusted to include the new field.

Reply:
PR opened : python-discord/site#277. Wdym by meta class?

Reply:
Here it is: python-discord/site#288